src/codegen/mod.rs

```diff
@@ -2499,6 +2499,14 @@ pub fn codegen(context: &mut BindgenContext) -> Vec<P<ast::Item>> {
             }
         }
 
+        if let Some(path) = context.options().emit_ir_graphviz.as_ref() {
+            println!("GRAPHVIZ CALLED!!!!");
+            match context.emit_ir_graphviz(path.clone()) {
+                Ok(()) => println!("Cool!"),
+                Err(e) => panic!("{}", e),
```

Comment:
Nitpick: it is probably OK to continue with bindings generation even if writing the dot file fails. We should log an `error!(...)` with a helpful diagnostic message, which will go to stderr and not get mixed in with the generated bindings.